package.json

```diff
@@ -30,18 +30,22 @@
     "socks": "^2.7.1"
   },
   "optionalDependencies": {
-    "@aws-sdk/credential-providers": "^3.186.0",
     "saslprep": "^1.0.3"
   },
   "peerDependencies": {
+    "@aws-sdk/credential-providers": "^3.186.0",
```

Comment:
We may want to bump this to a higher number: [v3.201.0](https://github.com/aws/aws-sdk-js-v3/blob/main/packages/credential-providers/CHANGELOG.md#32010-2022-11-01) matches our nodejs version support level

Reply:
Good point - updated to 3.201.0